rn/Teacher/package.json

```diff
@@ -28,11 +28,11 @@
     "fsevents": "^1.2.4"
   },
   "dependencies": {
-    "@react-native-community/art": "^1.2.0",
+    "@react-native-community/art": "instructure/art",
     "@react-native-community/async-storage": "^1.6.1",
     "@react-native-community/datetimepicker": "^3.0.0",
     "@react-native-community/picker": "^1.6.6",
-    "@react-native-community/segmented-control": "^2.1.1",
+    "@react-native-community/segmented-control": "instructure/segmented-control",
```

Comment:
`react-native-segmented-control/segmented-control` has the fix, I don't think we need our own fork of this.

Reply:
oh yeah, it's just not released yet. I couldn't figure out how to tell yarn to use "master".

Reply:
if you put in the repo name instead of the version number, it will use master, just like it did for the fork ;)